pom.xml

```
@@ -536,8 +536,8 @@
                 <configuration>
                     <skipCheckoutIfExists>false</skipCheckoutIfExists>
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
-                    <scmVersionType>tag</scmVersionType>
-                    <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+                    <scmVersionType>branch</scmVersionType>
+                    <scmVersion>CHK-3285-set-wallet-details-nullable</scmVersion>
```

```suggestion
                    <scmVersionType>tag</scmVersionType>
                    <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
```
TODO to be replaced after dependent pr merge